<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTrain</class>
 <widget class="QDialog" name="DialogTrain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Departure Point:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Destination Point:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Departure Time:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Number Seats:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Travel Duration:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Train Number:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Route:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLineEdit" name="IdLineEdit_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="DeparturePointLineEdit_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="DestinationPointLineEdit_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="DepartureTimeLineEdit_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="NumberSeatsLineEdit_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="TravelDurationLineEdit_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="TrainNumberLineEdit_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="NameLineEdit_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="RouteLineEdit_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Створити об'єкт</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
